WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -2629,17 +2632,14 @@ function WeakAuras.ReloadTriggerOptions(data)
   local iconCache = spellCache.Get();
   WeakAuras.EnsureOptions(id);
 
-  local trigger, untrigger, appendToTriggerPath, appendToUntriggerPath;
+  local appendToTriggerPath, appendToUntriggerPath;
```

Comment:
This is a bit more complex.

For multi-selection we construct the options twice:
* Once for each aura.
* Once for the whole selection

The whole selection options is then cleaned of all "functions", and those functions are replaced by functions that forward the call to each individual aura with some intelligent merging.

The trigger and untrigger values for the options creation are only used inside the functions, so while they are crucial for the per aura options creation, they aren't needed for creating options for the whole selection